ui/components/component-library/avatar-icon/avatar-icon.stories.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import {
   Size,
-  DISPLAY,
   AlignItems,
   BackgroundColor,
   IconColor,
   Color,
+  Display,
 } from '../../../helpers/constants/design-system';
 
 import Box from '../../ui/box/box';
```

Comment:
Could you update this storybook file to use the `Box` component from the component-library please?

Reply:
Sure @georgewrmarshall ,
Will do.

Reply:
Hey @georgewrmarshall ,
Probably, I am doing some mistake in trying to get the `Box` component from the `component-library`
OR, probably it is imported and exported in some other way.

Will try to figure out and report to you soon.

Reply:
Hey @georgewrmarshall ,
Tried to do that in the last commit.

Please look into it.